build/dependencies.yaml

```diff
@@ -30,7 +30,7 @@ dependencies:
     version: "2.31"
     refPaths:
     - path: test/utils/image/manifest.go
-      match: configs\[Agnhost\] = Config{promoterE2eRegistry, "agnhost", "\d+\.\d+"}
+      match: configs\[Agnhost\] = Config{list\.PromoterE2eRegistry, "agnhost", "\d+\.\d+"}
```

Comment:
This file just had to be updated to match the new line of code; no functional change.